### PR TITLE
css/example guides: isolate css changes to tables so they don't affect api docs

### PR DESCRIPTION
**Pull request by maiamcc**
Hello @nicks, @landism,

Please review the following commits I made in branch maiamcc/fix-centered-table:

f3217ece541badf1c3a76f3443c697e5d6e3ac15 (2020-03-11 18:12:24 -0400)
css/example guides: isolate css changes to tables so they don't affect api docs

Code review reminders, by giving a LGTM you attest that:

* Commits are adequately tested
* Code is easy to understand and conforms to style guides
* Incomplete code is marked with TODOs
* Code is suitably instrumented with logging and metrics

### CODE REVIEW

**Comment by maiamcc** [comment]
Previously, table styling changes for the example guides were scoped globally, leading to funkiness in the API reference:

Screen Shot 2020-03-10 at 12 01 14 PM

File: docs/example_static_html.md [hunk 1]
```diff
@@ -169,10 +180,24 @@ Tilt was able to update the container in less than a second!
 
 ### Final Score
 
-| Approach | Deploy Time |
-|---|---|
-| Naive | 1-2s |
-| With live_update | 0-1s |
+<table class="benchmark-report">
```
Comment (nicks) [inline docs/example_static_html.md]: there's a much easier way to do this, just prepend `{:.benchmark-report}` before the markdown table

Comment (nicks) [reply docs/example_static_html.md]: alternatively, just change the css to `table:not(.docutils)`

Comment (maiamcc) [reply docs/example_static_html.md]: 😅will do

**Pull request merged**